.github/depcheck.yml

```diff
@@ -11,6 +12,7 @@ go_modules:
 # List of Github repos to check for newer tags.
 github_repos:
   - github.com/google/dnsmasq_exporter v0.2.0
+  - github.com/justwatchcom/elasticsearch_exporter v1.1.0
```

Comment:
Ah, I should document this better. You only need to add an entry to `github_repos` if it's a dependency using `replace` in go.mod and we want to track updates to the official project and not the project that we replaced the module with. I'll see if I can remove this for you.